src/main/scala/org/ergoplatform/nodeView/history/storage/HistoryStorage.scala

```diff
@@ -34,16 +34,21 @@ class HistoryStorage(indexStore: Store, objectsStore: ObjectsStore, config: Cach
             Failure(e)
           }.toOption match {
             case Some(pm) =>
-              log.trace(s"Cache miss for existing modifier $key")
-              modifiersCache.put(key, pm)
+              log.trace(s"Cache miss for existing modifier $id")
+              modifiersCache.put(id, pm)
               Some(pm)
             case None => None
           }
         }
     }
   }
 
-  def getIndex(id: ByteArrayWrapper): Option[ByteArrayWrapper] = indexStore.get(id)
+  def getIndex(id: ByteArrayWrapper): Option[ByteArrayWrapper] = Option(indexCache.getIfPresent(id)).orElse {
```

Comment:
What does the term "index" mean here?

Reply:
A regular database index.

Reply:
Then indexCache as a "cache of a regular database index" is not very descriptive. Please use more descriptive name or write  comments.

Reply:
Wrote a description for the whole class